ja-jp/Content/index.htm

```diff
@@ -200,7 +219,7 @@
                                             <div class="dropdown" data-test="menuBarCategory" id="documentationMenu" data-mc-conditions="MultiProdPublish.DocMenu">
                                                 <button class="dropbtn">ドキュメント <i class="fa fa-caret-down"></i></button>
                                                 <div class="dropdown-content" data-test="menuBarItems">
-                                                    <div><a href="https://help.okta.com/ja-jp/Content/index.htm" data-test="menuBarLink">製品ドキュメント</a>
+                                                    <div><a href="../../en-us/Content/index.htm" data-test="menuBarLink">製品ドキュメント</a>
```

Comment:
This seems wrong, shouldn't it be a link to some page in `/ja-jp` folder?

Reply:
Wrong indeed. Previously, we didn't include this top-level `index.htm` file in the translation run, this file in JA was static. 
Meanwhile, our replacements pairs in `scripts/en2ja.json` do not have a match for this pattern (`/en-us/Content/`), which appears in most of the tile links in this file (linking out to the six product publications). And the top-level TOC looks suspicious, I'm investigating.
I can create a ticket to fix this properly, and meanwhile apply manual fixes to this file for this particular translation run?

Reply:
Sounds good.

For future reference if needed, here is the follow up jira: https://oktainc.atlassian.net/browse/OKTA-603889